src/internal/packager2/inspect.go

```diff
@@ -142,3 +145,178 @@ func PackageInspectManifests(ctx context.Context, pkgLayout *layout2.PackageLayo
 
 	return PackageInspectManifestResults{Resources: resources}, nil
 }
+
+type DevInspectManifestsOptions struct {
+	CreateSetVariables map[string]string
+	DeploySetVariables map[string]string
+	Flavor             string
+	KubeVersion        string
+}
+
+type DevInspectManifestResults struct {
+	Resources []Resource
+}
+
+// DevInspectManifests returns manifests and Helm chart manifests after templating
+func DevInspectManifests(ctx context.Context, packagePath string, opts DevInspectManifestsOptions) (DevInspectManifestResults, error) {
+	state, err := types.DefaultZarfState()
+	if err != nil {
+		return DevInspectManifestResults{}, err
+	}
+	pkg, err := layout.LoadPackageDefinition(ctx, packagePath, opts.Flavor, opts.CreateSetVariables)
+	if err != nil {
+		return DevInspectManifestResults{}, err
+	}
+	variableConfig := template.GetZarfVariableConfig(ctx)
+	variableConfig.SetConstants(pkg.Constants)
+	variableConfig.PopulateVariables(pkg.Variables, opts.DeploySetVariables)
+
+	tmpPackagePath, err := utils.MakeTempDir(config.CommonOptions.TempDirectory)
+	if err != nil {
+		return DevInspectManifestResults{}, err
+	}
+	defer os.RemoveAll(tmpPackagePath)
```

Comment:
💭 Future concern but is it bad if we silently fail cleanup here? My expectation is that this should be fine by default in CI, and capturing this error might add flakiness. Maybe we add a `tmp.IsCleanupFrom(ctx)` configuration flag or something

Reply:
In this case it was just something I forgot to do, but I believe we error on cleanup in other parts of the code. I think I'll go back today and turn the linter back on in packager2/ and fix situations like this.

Reply:
sgtm